addons/hr/static/src/models/thread/thread.js

```diff
@@ -0,0 +1,28 @@
+odoo.define('hr/static/src/models/thread/thread.js', function (require) {
+'use strict';
+
+const {
+    registerInstancePatchModel,
+} = require('mail/static/src/model/model_core.js');
+
+registerInstancePatchModel('mail.thread', 'hr/static/src/models/thread/thread.js', {
+    //--------------------------------------------------------------------------
+    // Private
+    //--------------------------------------------------------------------------
+
+    /**
+     * @override
+     * @returns {mail.partner[][]}
+     */
+    _getOrderedPreferredPartnerMentionsGroups() {
+        const preferredPartnersGroups = this._super();
+        const employees = this.env.models['hr.employee']
+            .all()
+            .map(employee => employee.partner)
+        ;
+        preferredPartnersGroups.splice(preferredPartnersGroups.length-1, 0, employees);
```

Comment:
I don't like `splice` because it changes in place.
Original method has `[main-list, extra-list]`. At a glance this mutates list so that it becomes `[main-list, employee-list, extra-list]`. Is it what we want? I thought the design was to keep only 2 lists?

Reply:
It's not linked to main & extra-list.
To build those lists, what is suggested here is to first look at followers of document, then all known partners.
BUT if hr is installed we need to search in employees before "all known partners".
The idea here is to have a list of group in which we search in order